IBuildaAZoo/IBuildaAZoo/Farm.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+namespace IBuildaAZoo
+{
+    public abstract class Farm
+    {
+        // Properties
+        public abstract string Wild { get;protected set: }
```

Comment:
`abstract` on automatic properties doesn't really make sense. Maybe `virtual`, but often even that isn't useful.